README.md

```diff
@@ -13,7 +13,10 @@ Start by adding the following snippet to your page template:
     Kumulos('init', {
         apiKey: 'YOUR_API_KEY',
         secretKey: 'YOUR_SECRET_KEY',
-        vapidPublicKey: 'YOUR_VAPID_KEY'
+        vapidPublicKey: 'YOUR_VAPID_KEY',
+        region: 'eu-central-1',
+        installId: 'original-visitor-1-id',
+        userId: 'optional-customer-1-id',
```

Comment:
For these, should we use naming consistent with Optimove SDK concepts, e.g. `originalVisitorId` and `customerId` ?

Reply:
(how we use them internally may be clearer if concepts map in 'public' API)

Reply:
Worth update in readme?